packages/tsconfig.json

```diff
@@ -23,6 +23,8 @@
     "target": "es5"
   },
   "exclude": [
-    "./angular"
+    "./angular",
```

Comment:
what's `./angular`?

Reply:
It was there already, I guess it can be removed